source/wp-content/themes/wporg-news-2021/sass/blocks/_code.scss

```diff
@@ -1,4 +1,5 @@
 // TODO: This can be removed when Gutenberg applies fontFamily correcly to .wp-block-code code (https://github.com/WordPress/gutenberg/issues/31135)
 .wp-block-code code {
 	font-family: var(--wp--custom--code--typography--font-family);
```

Comment:
Can we get rid of the Gutenbug workaround now?

Reply:
That was probably an artifact from forking Blockbase, so it's probably safe to remove if you don't see any obvious side-effects

Reply:
Technically this isn't doing anything because `--wp--custom--code--typography--font-family` is `monospace`, which is the default for `<code>`… but I think there's another (unrelated) bug here, because I don't see any of the block styles from `theme.json` applying.

Reply:
Found & fixed the issue I mentioned: https://github.com/WordPress/wporg-news-2021/commit/5f2db9d753bed8531a13b637522981465895b84f.